composition-js/src/validate.ts

```diff
@@ -623,11 +665,15 @@ class ValidationTraversal {
   private handleState(state: ValidationState) {
     debug.group(() => `Validation: ${this.stack.length + 1} open states. Validating ${state}`);
     const vertex = state.supergraphPath.tail;
-    const currentSources = state.currentSubgraphNames();
-    const previousSeenSources = this.previousVisits.getVertexState(vertex);
-    if (previousSeenSources) {
-      for (const previousSources of previousSeenSources) {
-        if (isSupersetOrEqual(currentSources, previousSources)) {
+
+    const currentVertexVisit: VertexVisit = {
+      subgraphs: state.currentSubgraphNames(),
+      overriddenLabels: state.selectedOverrideConditions
```

Comment:
nit: `overrideLabels` is the old name back from when this was a `Set`, would be good to update it since it's a `Map` now.